src/endpoint/s3/ops/s3_put_bucket_lifecycle.js

```diff
@@ -111,6 +112,11 @@ async function put_bucket_lifecycle(req) {
         if (rule.Status?.length !== 1) {
             dbg.error('Rule should have status', rule);
             throw new S3Error(S3Error.InvalidArgument);
+        } else if (rule.Status?.length === 1) {
```

Comment:
Something that is not clear from this logic is what if I have a rule without a Status, where will it fail?

Reply:
Added a condition to check for `Status` value `null` or `undefined`. Thanks